processor/src/main/java/org/infinispan/protostream/annotations/impl/processor/types/MirrorTypeFactory.java

```diff
@@ -73,9 +73,12 @@ public final class MirrorTypeFactory implements XTypeFactory {
 
    private final MirrorPrimitiveType doubleType;
 
+   private final boolean fullyQualifiedAnnotations;
+
    public MirrorTypeFactory(ProcessingEnvironment processingEnv) {
       elements = processingEnv.getElementUtils();
       types = processingEnv.getTypeUtils();
+      fullyQualifiedAnnotations = Boolean.parseBoolean(processingEnv.getOptions().get(AutoProtoSchemaBuilderAnnotationProcessor.FULLY_QUALIFIED_ANNOTATIONS));
```

Comment:
I guess this could be a parameter in the constructor.

Reply:
Since we are already passing in the `processingEnv` I thought it was a bit pointless.

Reply:
the field in `AutoProtoSchemaBuilderAnnotationProcessor` can be removed then :)

Reply:
Fixed